typescript/integration-tests/package.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "reality-capture-tests",
+	"name": "@itwin/reality-capture-tests",
```

Comment:
wont be publishing this

Reply:
added itwin prefix to sdk and sdk-node, the two to be published, added private to the rest